collector/lib/SysdigEventExtractor.h

```diff
@@ -130,6 +128,23 @@ class SysdigEventExtractor {
   FIELD_CSTR(evt_args, "evt.args");
   FIELD_RAW(ppid, "proc.ppid", int64_t);
 
+ public:
+  const uint32_t* get_uid(sinsp_evt* event) {
```

Comment:
A bit of a nit perhaps, but it might be worth defining another macro for extracting this kind of compound field:

```cpp
#define TINFO_FIELD_TYPE(name, field, type) \
    const type* get_##name(sinsp_evt* event) { \
         if (!event) return nullptr; \
         sinsp_threadinfo* tinfo = event->get_thread_info(true); \
         if (!tinfo) return nullptr; \
         return &tinfo->m_##field; \ 
    }

TINFO_FIELD_RAW(uid, "user.uid", uint32_t);
TINFO_FIELD_RAW(gid, "user.gid", uint32_t);
```